src/components/Avatar/Avatar.styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+import { Button } from '@ynput/ayon-react-components'
+
+export const Avatar = styled.div`
+    display: flex;
+    width: auto;
+    align-items: center;
+    justify-content: center;
+    align-self: center;
+    flex-direction: column;
+    .user-image {
+        border: none;
+    }
+`
+export const AvatarIcon = styled(Button)`
+    position: absolute;
+    top: 75%;
+    right: 5%;
```

Comment:
bottom: 0;
right: 0;

to get in bottom right corner

Reply:
Done.